src/videoCache.js

```diff
@@ -63,11 +63,11 @@ function wrapURI(uri, impUrl) {
 function toStorageRequest(bid, {index = auctionManager.index} = {}) {
   const vastValue = bid.vastXml ? bid.vastXml : wrapURI(bid.vastUrl, bid.vastImpUrl);
   const auction = index.getAuction(bid);
-
+  const ttlWithBuffer = Number(bid.ttl) + 15;
```

Comment:
Could we initialize the buffer amount to a descriptively named const?

Reply:
Yeah